File: Nodejs/bitcode/bitFuncs.js
// Sets the bit at pos in the byteeger n.
exports.setBit = function(n, pos) {
    n |= (1 << pos);
    return n;
}

// Clears the bit at pos in n.
exports.clearBit = function(n, pos) {
    let mask = ~(1 << pos);
    n &= mask;
    return n;
}

exports.hasBit = function(n, pos) {
    let val = n & (1 << pos);
    return (val > 0);
}

File: Python/bitFuncs.py
# Sets the bit at pos in the byteeger n.
def setBit(n, pos):
    mask = (1 << pos)
    return (n | mask)

# Clears the bit at pos in n.
def clearBit(n, pos):
    mask = ~(1 << pos)
    return (n & mask)

def hasBit(n, pos):
    val = n & (1 << pos)
    return (val > 0)